apps/system/js/app_chrome.js

```diff
@@ -741,6 +751,7 @@
     if (!this._themeChanged) {
       this.setThemeColor('');
     }
+    this.setSiteIcon();
```

Comment:
BTW, what happens if the XHR for the manifest doesn't finish until after loadend? Do we just ignore the icon?